### PR TITLE
 Cleaning out some big files slowly.

### PR DESCRIPTION
**Pull request by DEV-DIBSTER**
- Trying to make the bot a tiny more readable.

### CODE REVIEW

**Comment by domin-mnd** [comment]
To make it more readable, please remove hardcode in each of the bot file e.g. move Discord IDs to json/js object files.

Also please do not use capital case for variables. This is syntax used for declaring classes and types, variables follow camelCase in most cases. If you're really into changing the case, consider on UPPER_CASE for constant variables.

File: bot/discord/events/ready.js [hunk 1]
```diff
@@ -68,18 +67,15 @@ module.exports = async (client) => {
...
+    //Something to do with invites.
     global.invites = {};
     client.guilds.cache.forEach((g) => {
         g.fetchInvites().then((guildInvites) => {
             invites[g.id] = guildInvites;
         });
     });
 
-    //Node status channel embed
+    //Node Status Channel Embed.
```
Comment (WilliamDavidHarrison) [inline bot/discord/events/ready.js:78]:
```suggestion
    // Node status embed
```

File: bot/discord/events/ready.js [hunk 2]
```diff
@@ -68,18 +67,15 @@ module.exports = async (client) => {
         });
     }, 30000);
 
-    // Voice-Channels:
-    client.pvc = new Discord.Collection();
-
-    // end of Voice-Channels
+    //Something to do with invites.
```
Comment (WilliamDavidHarrison) [inline bot/discord/events/ready.js:70]:
```suggestion
    // Fetch and store guild invites
```

File: bot/discord/events/ready.js [hunk 3]
```diff
@@ -16,7 +16,6 @@ module.exports = async (client) => {
     checkNicks();
 
     console.log(chalk.magenta("[DISCORD] ") + chalk.green(client.user.username + " has logged in!"));
-    //getUsers()
 
     //Check make sure create account channels are closed after a hour
```
Comment (WilliamDavidHarrison) [inline bot/discord/events/ready.js:20]:
```suggestion
    // Close create account channels after a hour
```

File: bot/discord/events/message.js [hunk 4]
```diff
@@ -120,7 +141,7 @@ module.exports = async (client, message) => {
         }
     }
 
-    //After command remove all clone traces
+    // After command remove all clone traces
```
Comment (WilliamDavidHarrison) [inline bot/discord/events/message.js:144]:
```suggestion
    // Remove all clone traces after running command
```

File: bot/discord/events/message.js [hunk 5]
```diff
@@ -70,27 +91,27 @@ module.exports = async (client, message) => {
...
+        ) return;
 
         if (
             sudo.get(message.member.id) &&
             message.member.roles.cache.find((r) => r.id === "898041747597295667") &&
             args[0] != "sudo"
         ) {
-            //Doubble check the user is deffinaly allowd to use this command
-            actualExecutorId = JSON.parse(JSON.stringify({ a: message.member.id })).a; // Deep clone actual sender user ID
+            // Double check the user is allowed to use this command.
```
Comment (WilliamDavidHarrison) [inline bot/discord/events/message.js:104]:
```suggestion
            // Double check the user is allowed to use this command
```

File: bot/discord/events/message.js [hunk 6]
```diff
@@ -34,20 +50,25 @@ module.exports = async (client, message) => {
                 message.channel.send(`\`\`\`${err.message}\`\`\``);
             }
         };
-    }
+    };
+
+    if (message.author.bot) return; // To stop bots from creating accounts, tickets and more.
```
Comment (WilliamDavidHarrison) [inline bot/discord/events/message.js:55]:
```suggestion
    if (message.author.bot) return; // Stop bots from running commands
```

File: bot/discord/events/message.js [hunk 7]
```diff
@@ -34,20 +50,25 @@ module.exports = async (client, message) => {
...
+    if (message.channel.type === "dm") return; // Stops commands working in dms
```
Comment (WilliamDavidHarrison) [inline bot/discord/events/message.js:56]:
```suggestion
    if (message.channel.type === "dm") return; // Stop commands in DMs
```

File: bot/discord/events/message.js [hunk 8]
```diff
@@ -22,8 +29,17 @@ module.exports = async (client, message) => {
...
+    const dmAllowedUsers = [
+        "137624084572798976", // Dan
+        "853158265466257448", // William
+        "757296951925538856", // DIBSTER
+        "459025800633647116"  // AVIXITY
+    ];
+
     if (message.channel.type === "dm") {
-        if (message.author.id === "137624084572798976" || message.author.id === "853158265466257448" || message.author.id === "757296951925538856" || message.author.id === "459025800633647116") {
+        // Checks if the member is one of the members allowed. If it is, allows the member to send messages on behalf of the bot.
```
Comment (WilliamDavidHarrison) [inline bot/discord/events/message.js:41]:
```suggestion
        // Allow users to send messages on behalf of the bot if they are allowed
```

File: bot/discord/events/message.js [hunk 9]
```diff
@@ -10,10 +11,16 @@ module.exports = async (client, message) => {
             .setTimestamp(new Date());
 
         client.channels.cache.get(config.DiscordBot.oLogs).send(embed);
-    }
-    //Auto reactions on suggestions
-    if (message.channel.id === "980595293768802327" || message.channel.id === "976371313901965373") {
-        if (!message.content.includes(">")) {
+    };
+
+    // Auto reactions on suggestions channels
```
Comment (WilliamDavidHarrison) [inline bot/discord/events/message.js:16]:
```suggestion
    // Suggestions channels reactions
```

File: bot/discord/events/message.js [hunk 10]
```diff
@@ -1,4 +1,5 @@
 module.exports = async (client, message) => {
+    // Checks the amount of member pinged in a message.
```
Comment (WilliamDavidHarrison) [inline bot/discord/events/message.js:2]:
```suggestion
    // Ban members which ping 20+ users
```

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]

**Review by WilliamDavidHarrison — APPROVED** [review]

**Pull request merged**